WordPress/Classes/Services/BlogService.m

```diff
@@ -356,6 +356,21 @@ - (NSArray *)blogsForAllAccounts
     return [self blogsWithPredicate:nil];
 }
 
+- (NSDictionary *)blogsForAllAccountsById
+{
+    NSMutableDictionary *blogMap = [NSMutableDictionary dictionary];
```

Comment:
Thanks for the detailed reply Diego!. I usually go with the `[ @[ ] mutableCopy` shorthand, whenever i do have initial values (so that you don't need to insert the objects, one by one). But in this particular case i'd defer in favor of the constructor, for type safety checks, as Diego pointed out.

Thank you both :smile: